homeassistant/components/control4/light.py

```diff
@@ -175,14 +94,17 @@ def create_api_object(self):
     @property
     def is_on(self):
         """Return whether this light is on or off."""
-        return self.coordinator.data[self._idx]["value"] > 0
+        if "LIGHT_LEVEL" in self.extra_state_attributes:
+            return self.extra_state_attributes["LIGHT_LEVEL"] > 0
+        if "LIGHT_STATE" in self.extra_state_attributes:
+            return self.extra_state_attributes["LIGHT_STATE"] > 0
+        if "CURRENT_POWER" in self.extra_state_attributes:
+            return self.extra_state_attributes["CURRENT_POWER"] > 0
 
     @property
     def brightness(self):
         """Return the brightness of this light between 0..255."""
-        if self._is_dimmer:
-            return round(self.coordinator.data[self._idx]["value"] * 2.55)
-        return None
+        return self.extra_state_attributes["LIGHT_LEVEL"] * 2.55
```

Comment:
We shouldn't access `extra_state_attributes`.  We should access the data that is being used to create it instead.